ext/reflection/tests/bug74351.phpt

```diff
@@ -0,0 +1,43 @@
+--TEST--
+Bug #74351 (Wrong reflection on variadic array functions)
+--FILE--
+<?php
+$functions = [
+    'array_diff',
```

Comment:
@Ocramius thanks, I've just added. BTW, we have such kind of tests here already https://github.com/php/php-src/blob/master/ext/reflection/tests/ReflectionFunction_isVariadic_basic.phpt

but I've added it to make it more implicit.